python/_restclient/tests/test_utilities.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+import pytest
+from hydrotools._restclient import utilities
+
+
+@pytest.fixture
+def alias_fixture():
+    return utilities.Alias("key", "value")
+
+
+test_instance_args = [
+    ("this", "that"),
+    (1, 2),
+    (1.0, 2.0),
+    (lambda x: x, (1, 2)),
+    ((1, 2), "key"),
+]
+
+
+@pytest.mark.parametrize("key,value", test_instance_args)
+def test_instance_alias(key, value):
+    """ Test inputs that can instantiate the class"""
+    utilities.Alias(key, value)
+
+
+def test_cannot_alter_alias_key(alias_fixture):
+    from dataclasses import FrozenInstanceError
+
+    with pytest.raises(FrozenInstanceError):
+        alias_fixture.key = "that"
+
+
+def test_cannot_alter_alias_key(alias_fixture):
```

Comment:
duplicate of previous test

Reply:
Actually, name is duplicated, function is not.  This actually tests changing the value, not the key.  Rename to `test_cannot_alter_alias_value`

Reply:
Nice catch!